pre_commit_hooks/check_merge_conflict.py

```diff
@@ -12,23 +12,26 @@
 WARNING_MSG = 'Merge conflict string "{0}" found in {1}:{2}'
 
 
-def is_in_merge():
+def is_in_merge(args):
     return (
-        os.path.exists(os.path.join('.git', 'MERGE_MSG')) and
         (
-            os.path.exists(os.path.join('.git', 'MERGE_HEAD')) or
-            os.path.exists(os.path.join('.git', 'rebase-apply')) or
-            os.path.exists(os.path.join('.git', 'rebase-merge'))
-        )
+            os.path.exists(os.path.join('.git', 'MERGE_MSG')) and
+            (
+                os.path.exists(os.path.join('.git', 'MERGE_HEAD')) or
+                os.path.exists(os.path.join('.git', 'rebase-apply')) or
+                os.path.exists(os.path.join('.git', 'rebase-merge'))
+            )
+        ) or args.assume_in_conflict
     )
 
 
 def detect_merge_conflict(argv=None):
     parser = argparse.ArgumentParser()
     parser.add_argument('filenames', nargs='*')
+    parser.add_argument('--assume-in-conflict', action='store_true')
     args = parser.parse_args(argv)
 
-    if not is_in_merge():
+    if not is_in_merge(args):
```

Comment:
probably can leave `is_in_merge` alone and just do:

```python
if not is_in_merge() and not args.assume_in_merge:
    return 0
```

(oops, sorry, let's also change the argument to `--assume-in-merge` now that I'm reading the code /o\)

Reply:
fixed